packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -283,90 +279,51 @@ export class PluginLedgerConnectorQuorum
     req: InvokeContractJsonObjectV1Request,
   ): Promise<any> {
     const fnTag = `${this.className}#invokeContractNoKeychain()`;
-    if (req.contractJSON != undefined) {
-      const networkId = await this.web3.eth.net.getId();
-      const contractJSON = req.contractJSON as any;
-      const contract = req;
-      if (
-        contractJSON.networks === undefined ||
-        contractJSON.networks[networkId] === undefined ||
-        contractJSON.networks[networkId].address === undefined
-      ) {
-        if (isWeb3SigningCredentialNone(req.signingCredential)) {
-          throw new Error(`${fnTag} Cannot deploy contract with pre-signed TX`);
-        } else {
-          throw new Error(
-            `${fnTag} Unsupported invocation type ${req.invocationType}`,
-          );
-        }
-      } else if (
-        req.contractJSON == undefined &&
-        req.contractAbi == undefined &&
-        req.contractAddress == undefined
-      ) {
+    if (req.contractJSON == undefined) {
```

Comment:
@elenaizaguirre Check falsy-ness with the exclamation mark, it's more robust.

Reply:
Done!